src/main/java/competition/subsystems/internalconveyor/commands/ManualReverseCommand.java

```diff
@@ -0,0 +1,28 @@
+package competition.subsystems.internalconveyor.commands;
+import com.google.inject.Inject;
+
+import competition.operator_interface.OperatorInterface;
+import competition.subsystems.internalconveyor.KickerSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class ManualReverseCommand extends BaseCommand{
```

Comment:
With this name it's not clear what we are reversing. I suggest `KickerReverseCommand`